docs/dice_verification.md

```diff
@@ -1,40 +1,206 @@
+<kbd><img src="img/dicedoc/seedtool_zpub.png"></kbd><br><br>
+Compare to zpub in Sparrow:<br>
+<kbd><img src="img/dicedoc/sparrow_zpub.png"></kbd>
+<br>
+
+Zpub is the same as shown in SeedSigner, Sparrow and Ian Colemand tool.
+<br><br>
+**Addresses:**<br>
+Scroll down a little bit where the receive addresses are shown and compare to the ones generated in Sparrow ('Addresses' tab of the wallet):<br>
+<kbd><img src="img/dicedoc/seedtool_addresses.png"></kbd>
+<br>
+
+Check that the receive addresses all match.<br><br>
+To verify the change addresses change the 'Receive/Change' dropdown box to '1 (Change)' (1):
+<kbd><img src="img/dicedoc/seedtool_change_addresses_1.png"></kbd><br><br>
+Compare the change addresses to the ones generated in Sparrow ('Addresses' tab of the wallet):
+<kbd><img src="img/dicedoc/seedtool_change_addresses_2.png"></kbd>
+<br>
+
+Check that the change addresses all match.
+<br><br><br>
+
+## 50 Dice Rolls / 12 Seed Words Example
+
+SeedSigner supports the creation of mnenomic seeds both with 12 or 24 seed words corresponding to 50 or 99 dice rolls. Below are some example result values for using 50 dice rolls only.<br><br>
+All the steps shown can be executed the same way, just select the '12 words (50 rolls)' option in SeedSigner and change the 'Mnenomic Length' dropdown boxes in both web tools to '12 Words'.<br><br>
+
+50 dice roll results as an example for a 12 words seed:<br>
+> 65515223131652132161133154444123616466443112153441
+
+The corresponding 12 seed words are:<br>
+> hole luggage safe present express tragic orbit shed switch metal identify path
+
+Fingerprint:<br>
+> 8d9cced8
+
+Zpub:<br>
+> zpub6qf9ziL759pzyhKMWaPfNSiCETkoA6oq3fbCDvXqcURiMtPnkEg3nH93W5mrSkvGPoJC9xTYZheYDsYoiYc5AkSk9iY3DkCJHkFgHMdijW6
+
+Addresses:<br>
+Receive:<br>
+> bc1q00lln3r4mt4uwvg7mxv96xgpewauwmggkex2ff<br>
+> bc1q0jj2cv965f3642mv4lgq5za80jtfpkd0jhjefr<br>
+> bc1qpecssejm2678v0rknk9tsxd5fsshezfr0vr5m5<br>
+> bc1qcsl37xn5rkfz8qhcfwq5u52acxecyyfz0kv4gh<br>
+> bc1q8ehx53re0wck4m9tzek8mlnctp2ztm7jq94zm4<br>
+> ...<br>
+
+Change:<br>
+> bc1qz0ckhg3m349qpmweyn5v6r6tvx2wfw4nv8h75q<br>
+> bc1qme5tu2t424ws3z69u445q0yw88vpc7fwygra4r<br>
+> bc1qznuyuc087ky7fhv4nvlmll4p586ks4ggcyt36d<br>
+> bc1qjcqxv22j0g00pehruwwh34sw5znu6vp3myaspy<br>
+> bc1q6dpfl7czd22wt0max6p09vr6lvvpag7xw9u8lc<br>
+> ...
+
+<br><br>
+## Conclusion
+
+What did we achieve now?<br>
+
+We created a dice-based seed in SeedSigner and set up a wallet using this seed in Sparrow wallet (as this is what a seed is used for).<br><br>
+We double-checked in two different web tools implementing different methods for seed creation that what SeedSigner generates perfectly matches up with what the other tools calculate based on the same dice entropy used.<br>
+
+So congratulations if the Fingerprints, Zpubs and Addresses all match up in your example so you can be much more confident that nothing is wrong with your generated seed.
```

Comment:
"Fingerprints, Zpubs and Addresses" becomes '"fingerprints", "zpubs", and "addresses"'